### DISCUSSION TITLE
Sell gui size

### DISCUSSION THREAD

**Discussion by Jirka135** [Zrips/CMI#7233] [Feature Requests]
Hi, It would be nice to add some setting for /sell gui size.
Thanks.

**Comment by mrfloris** [comment]
Yeah it is so weird, for some reason I vividly remember that this was possible because I thought in the past that I had to resize it from 9 to 4 rows to cover all the inventory slots plus hotbar. But I searched and can't find it. Maybe that was only for dispose or something. 

I wouldn't mind limiting this on some pvp servers to add to the aniety of the fight, by not allowing /trash and /sell to have more than 9 items.. so they can't use macro sell all and auto buy, but that they have to be more selective.  To just name one example.
